config/default/ActivityMonitoringService.conf.php

```diff
@@ -10,4 +10,5 @@
 return new ActivityMonitoringService([
     ActivityMonitoringService::OPTION_ACTIVE_USER_THRESHOLD => 300,
     ActivityMonitoringService::OPTION_COMPLETED_ASSESSMENTS_AUTO_REFRESH => 30,
+    ActivityMonitoringService::OPTION_ASSESSMENT_ACTIVITY_AUTO_REFRESH => 60,
```

Comment:
I couldn't find a way to update a chart in c3 without a url that didn't cause an error that aleh previously pointed out. So, I reverted to using two data calls: one for assessment activity data, and the other for completed assessment data. I thought this ok because these two calls are fundamentally different. That is, assessment activity data retrieves it's data from the ActivityMonitoringService. And completed assessment data retrieves from the LoggerService. So, two auto refresh options are now used.